jsk_fetch_robot/jsk_fetch_gazebo_demo/README.md

```diff
@@ -0,0 +1,57 @@
+jsk_fetch_gazebo_demo
+=====================
+
+- This demo is roseus version of [fetch\_gazebo\_demo](https://github.com/fetchrobotics/fetch_gazebo/tree/gazebo9/fetch_gazebo_demo).
+- You can learn how to use roseus and how ROS components are executed. (e.g. pointcloud, navigation, motion planning, ... etc)
+
+## Usage
+- Build this package like [jsk\_fetch\_startup README](https://github.com/jsk-ros-pkg/jsk_robot/tree/master/jsk_fetch_robot#setup-environment)
+```bash
+mkdir -p catkin_ws/src
+cd catkin_ws/src
+git clone https://github.com/jsk-ros-pkg/jsk_robot.git
+rosdep install --from-paths . --ignore-src -y -r
+cd ..
+catkin build jsk_fetch_gazebo_demo
```

Comment:
```suggestion
catkin config
catkin build jsk_fetch_gazebo_demo
```

Reply:
Sorry, I don't understand the role of `catkin config`
Do you mean `catkin config -DCMAKE_BUILD_TYPE=Release` ?

Reply:
can we build the workspace wothout catkin init or catkin config as this readme?

Reply:
> can we build the workspace wothout catkin init or catkin config as this readme?

I think we can build the workspace, but it's no problem to add `catkin config`

Reply:
i didnt't know that we can build specific package in the workspace without catkin init